vsphere/resource_vsphere_revert_snapshot.go

```diff
@@ -0,0 +1,71 @@
+package vsphere
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceVSphereRevertSnapshot() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVSphereSnapshotRevert,
+		Delete: resourceVSphereSnapshotDummyDelete,
+		Read:   resourceVSphereSnapshotDummyRead,
+
+		Schema: map[string]*schema.Schema{
+			"datacenter": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+			"vm_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"snapshot_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"suppress_power_on": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceVSphereSnapshotRevert(d *schema.ResourceData, meta interface{}) error {
+	vm, err := findVM(d, meta)
+	if err != nil {
+		return fmt.Errorf("Error while getting the VirtualMachine :%s", err)
+	}
+	task, err := vm.RevertToSnapshot(context.TODO(), d.Get("snapshot_id").(string), d.Get("suppress_power_on").(bool))
+	if err != nil {
+		log.Printf("[ERROR] Error While Creating the Task for Revert Snapshot: %v", err)
```

Comment:
A note on logs: Terraform currently does nothing with `INFO` and `ERROR` messages that is anything more meaningful than `DEBUG`... can all of these messages be changed to `DEBUG`?